src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -455,3 +464,112 @@ def resume_privacy_request(
     ).submit(from_webhook=webhook)
 
     return privacy_request
+
+
+def review_privacy_request(
+    db: Session, cache: FidesopsRedis, request_ids: List[str], process_request: Callable
```

Comment:
I like the use of `process_request` here! Nice way to reuse the main error handling workflow.

Reply:
oh thank you!